cpp/src/strings/convert/convert_datetime.cu

```diff
@@ -198,8 +211,9 @@ struct parse_datetime {
     auto ptr    = d_string.data();
     auto length = d_string.size_bytes();
```

Comment:
```suggestion
    auto const length = d_string.size_bytes();
```

Reply:
This is a working variable that is modified inside the for-loop just that starts just below this line.